docs/macros/short-bind.md

```diff
@@ -0,0 +1,96 @@
+# shortBind
+
+<StabilityLevel level="experimental" />
+
+A shorthand for binding prop with the same data name.
+
+`:value` -> `:value="value"`
+
+If you have any questions about this feature, you can comment on [RFC Discussion](https://github.com/vuejs/rfcs/discussions/405).
+
+|   Features   |     Supported      |
+| :----------: | :----------------: |
+|    Vue 3     | :white_check_mark: |
+|    Nuxt 3    | :white_check_mark: |
+|    Vue 2     |        :x:         |
+| Volar Plugin | :white_check_mark: |
+
+## Setup
```

Comment:
We can remove this chapter now, see https://github.com/vue-macros/vue-macros/blob/main/packages/boolean-prop/src/index.ts